clj-refactor.el

```diff
@@ -560,11 +560,28 @@ to us to make sure it's nicely indented."
     (goto-char (point-min))
     (search-forward package nil t)))
 
+(defun cljr--find-source-ns-of-test-ns (test-ns test-file)
+  (let* ((ns-chunks (split-string test-ns "[.]" t))
+         (test-name (car (last ns-chunks)))
+         (src-dir-name (s-replace "test/" "src/" (file-name-directory test-file)))
+         (replace-underscore (-partial 's-replace "_" "-"))
+         (src-ns (car (--filter (or (string-prefix-p it test-name)
+                                    (string-suffix-p it test-name))
+                                ;; Cannot find dash.el's -compose?
```

Comment:
This comment seems unnecessary.  Either solve the problem (are we using an old version of dash?) or proceed with confidence like everything is honky dory :p